packages/expo-sqlite/ios/Exceptions.swift

```diff
@@ -51,3 +51,13 @@ internal class InvalidArgumentsException: Exception {
     "args must be an array"
   }
 }
+
+internal class SQLiteErrorException: GenericException<String> {
+  override var code: String {
+    "E_SQLITE"
```

Comment:
If you for some reasons don't want to use the automatic error code, we start all error codes from `ERR`. Also, I would make it a bit more descriptive - we can't do much, but maybe `ERR_INTERNAL_SQLITE_ERROR`? It would at least indicate it's coming from the sqlite3 lib.
```suggestion
    "ERR_INTERNAL_SQLITE_ERROR"
```